psst-gui/src/controller/playback.rs

```diff
@@ -310,6 +315,12 @@ impl PlaybackController {
             },
         }));
     }
+
+    fn update_lyrics(&self, ctx: &mut EventCtx, data: &AppState, now_playing: &NowPlaying) {
+        if matches!(data.nav, Nav::Lyrics) {
+            ctx.submit_command(lyrics::SHOW_LYRICS.with(now_playing.clone()));
+        }
+    }
```

Comment:
Not sure if this is the best place for this but adding it to the PlaybackController allows us to link the lyrics with the song.

Reply:
I think this is good!